README.md

```diff
@@ -147,6 +146,13 @@ While your workspace itself is a clone of [MovingBlocks/Terasology](https://gith
 Accordingly, if you want to contribute to modules, you'll need to navigate into the respective subdirectory and work with Git from in there.
 Any Git commands executed in your workspace root will target [MovingBlocks/Terasology](https://github.com/MovingBlocks/Terasology).
 
+For a quick setup execute the following:
+```
+./groovyw module init iota
+./gradlew game
+
+```
+
```

Comment:
moved it up a little. for the command, this is developers - so i really expect them to know the dot or not dot. only in cmd it does not work. it works in powershell, git bash, linux, mac.

Reply:
i wanted to make the readme shorter, not longer. maybe remove the contributor quickstart guide. because i am always searching for this 2 commands to setup the workspace i wanted to have them in the readme, it is hidden too well, as well it is hidden too well that it works with new java.